src/lib/Input.svelte

```diff
@@ -24,8 +34,37 @@
         messageReceived = false;
     };
 
+    /**
+     * Returns a boolean indicating whether the innerText contains letters.
+     *
+     * @returns {boolean} Whether the innerText contains letters.
+     */
+    function innerTextContainsLetters(): boolean {
```

Comment:
Awesome to see your a regex wiz! This is just a nitpick, but for the sake of brevity, I might rewrite a function like this as:

```ts
const innerTextContainsLetters = () => Boolean(innerText.match(/[a-z]/i))
```

though I might be a little too-obsessed with writing one-liners...

Similar to above, this is a perfectly named function (and not exposed outside of this file) so the comment is probably unnecessary

Reply:
Yep that sounds good!